.github/workflows/patch_update_main.yml

```diff
@@ -16,6 +16,7 @@ jobs:
     - name: Checkout your repository
       uses: actions/checkout@v4
       with:
+        token: ${{ secrets.PAT }}
```

Comment:
Maybe add a comment on why using this token over the runner provided one.

Reply:
Done, thanks!